### ISSUE TITLE
Code very slow

### ISSUE THREAD

**Issue by LilianLaporte** [danielgordon10/re3-pytorch#8]
Hello,

When I run the image_demo.py, the creation of the instance [tracker](https://github.com/danielgordon10/re3-pytorch/blob/04e10d2a17a81db7cd40de3f73135b0cfe0da10e/demo/image_demo.py#L19) and of the [first call of track](https://github.com/danielgordon10/re3-pytorch/blob/04e10d2a17a81db7cd40de3f73135b0cfe0da10e/demo/image_demo.py#L22) take a lot of time (several minutes). If I am using the small weights it does not change the problem. On another hand, the version with [tensorflow](https://github.com/danielgordon10/re3-tensorflow/tree/master) is really fast and works properly. Is it normal if it is that slow or should it be fast?

NB: my GPU is a NVIDIA A4000

**Comment by danielgordon10** [comment]
No, it should be fast. Can you add some timing statements to figure out which line is causing the slowdown? Maybe torch is trying to set up some optimization under the hood.
